src/tmpfiles/tmpfiles.c

```diff
@@ -2307,17 +2408,21 @@ static int clean_item_instance(Item *i, const char* instance) {
                         sx.stx_ino != ps.st_ino;
         }
 
-        log_debug("Cleanup threshold for %s \"%s\" is %s",
+        if (age_by_format(i->age_by, &age_by) < 0)
+                return log_oom();
+
+        log_debug("Cleanup threshold for %s \"%s\" is %s %s",
                   mountpoint ? "mount point" : "directory",
                   instance,
-                  format_timestamp_style(timestamp, sizeof(timestamp), cutoff, TIMESTAMP_US));
+                  format_timestamp_style(timestamp, sizeof(timestamp), cutoff, TIMESTAMP_US),
+                  age_by);
```

Comment:
please enclose the formatting in 

```c
if (DEBUG_LOGGING)  {
         _cleanup_free_ char *age_by = NULL;
         if (age_by_format(…

         log_debug(…
}
```

i.e. do the formatting work only if debug logging s actually on